### PR TITLE
[2.19.x] DDF-UI-112 allow intersects query for line

### PR DESCRIPTION
**Pull request by cassandrabailey293**
#### What does this PR do?
Allows a line to have a zero-buffer. Solves https://github.com/codice/ddf-ui/issues/112

#### Who is reviewing it? 

@hayleynorton @andrewzimmer @zta6 
#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining 

#### How should this be tested?

- Advanced search - anyGeo
- Select Line
- Draw a line and verify that it doesn't automatically change the buffer width to 1
- Verify that you can search with nothing (and/or 0) in the buffer width input, and that the outgoing cql uses INTERSECTS instead of DWITHIN
- Verify that searching with a buffer width greater than 1 meter is allowed and that the outgoing cql uses DWITHIN
- Verify that searching with a buffer width of greater than 0 but less than 1 meter displays an inline error message "Buffer width must be 0 or at least 1 meter".  similar message should display when clicking search (the search should be prevented). Error message should change based on which units you have selected
- Verify no regression with Line searches

#### What are the relevant tickets?
Fixes:  https://github.com/codice/ddf-ui/issues/112

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/location/base.line.js [hunk 1]
```diff
@@ -116,13 +116,13 @@ const BaseLine = props => {
...
       )
       if (props.drawing) {
-        if (
-          geometryKey.includes('line') &&
-          (lineWidth === undefined || Number(lineWidth) <= 0)
-        ) {
-          setState({ [widthKey]: 1 })
-          setBufferError(initialErrorState)
-        }
+        // if (
```
Comment (zta6) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/location/base.line.js]: ❓ should we just remove this code?

Comment (cassandrabailey293) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/location/base.line.js]: woops. yes :0)

**Comment by andrewzimmer** [comment]
Hero Successful 🎉 
- verified lines draw correctly on map
- verified line with no buffer executed as intersects search and returns results
- verified no regression

**Comment by cassandrabailey293** [comment]
build now

**Pull request merged**